static/js/place/place.ts

```diff
@@ -40,6 +40,9 @@ const Y_SCROLL_WINDOW_BREAKPOINT = 992;
 // Margin to apply to the fixed sidebar top.
 const Y_SCROLL_MARGIN = 100;
 const placeTypesWithChoropleth = new Set(["Country", "State", "County"]);
+const CATEGORY_REDIRECT = {
+  Climate: "Environment",
```

Comment:
nit: can we do the redirect in flask? it'll be much faster (no need to download and execute js).

Reply:
done

Reply:
perfect - lgtm